code/modules/reagents/chemistry/reagents/drinks_reagents.dm

```diff
@@ -173,32 +173,36 @@
 	id = "banana"
 	description = "The raw essence of a banana."
 	color = "#F6F834"
+	process_flags = ORGANIC | SYNTHETIC
 	drink_icon = "banana"
 	drink_name = "Glass of banana juice"
 	drink_desc = "The raw essence of a banana. HONK"
 	taste_description = "banana juice"
 
 /datum/reagent/consumable/drink/banana/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
+	var/is_robot = ((process_flags & SYNTHETIC) > 0)
```

Comment:
```suggestion
```
Not needed here. This is used in omnizine because there is a synthetic subtype (it cuts down on duplicated code).